rules/keep-tasks-handy-for-calls/rule.md

```diff
@@ -0,0 +1,32 @@
+---
+type: rule
+title: Do you always keep your task list handy for when the boss calls?
+uri: keep-tasks-handy-for-calls
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy/
+created: 2024-05-22T13:55:00.000Z
+guid: 8f304a6b-8345-4e3b-ac6f-259e028bb0c2
+
```

Comment:
Hi @Tanya-Leahy 

1. Can you find a rule about having a list of tasks in priority and add here as Related?

Reply:
Done - linked back and forth from work-in-order-of-importance-aka-priorities.